ome/figure
# Static demo grunt task

Adding a grunt task to build the static demo index.html and figure.js files, ready to be copied over to the gh-pages demo/ folder.

See README for more info.

The result of running the ``` grunt demo ``` task has been copied to https://github.com/ome/figure/pull/110, tested and merged.

To test the grunt task you'll need to 
 - Install Node from https://nodejs.org/en/
 - Clone the figure repo and checkout the ``` static_demo_grunt_task``` branch
 - then:
```
$ cd figure
$ npm install        # should install all dependencies
$ grunt demo
```
Should see something like
```
$ grunt demo
Running "concat:js" (concat) task
File static/figure/js/figure.js created.

Running "replace:dist" (replace) task
>> 92 replacements in 2 files.

Running "copy:main" (copy) task
Created 28 directories, copied 98 files

Done, without errors.
```
Then you should be able to do:
```
    $ cd demo/
    $ python -m SimpleHTTPServer
```
Go to http://localhost:8000/ to test it.
